packages/dds/tree/src/simple-tree/simpleSchema.ts

```diff
@@ -162,6 +184,20 @@ export type SimpleNodeSchema =
 	| SimpleObjectNodeSchema
 	| SimpleRecordNodeSchema;
 
+/**
+ * Information about allowed types under a field.
+ *
+ * @alpha
+ * @sealed
+ */
+export interface AllowedTypeInfo {
```

Comment:
We already have many types which provide information about allowed types, This one's name should di8fferentiate how its intended to be used. `SimpleAllowedTypes` would be a better name.